packages/oidc-client/src/lib/token.types.ts

```diff
@@ -0,0 +1,17 @@
+import { OidcConfig } from './config.types.js';
+
+export interface TokenExchangeResponse {
+  access_token: string;
+  id_token?: string;
+  refresh_token?: string;
+  expires_in?: number;
+  scope?: string;
+  token_type?: string;
+}
+
+export interface TokenRequestOptions {
+  code: string;
+  config: OidcConfig;
+  endpoint: string;
+  verifier?: string;
+}
```

Comment:
Should these interfaces be in `exchange.types.ts`?

Reply:
Yeah, you're right. I'll do that now.